test/validators.js

```diff
@@ -5935,12 +5939,12 @@ describe('Validators', () => {
           '5501599623874',
           '+55012962308',
           '+55 015 1234-3214',
-          '+55 11 91431-4567',
```

Comment:
Why were these removed, did you introduce a regression? I can't see it from the code change...

Reply:
Hey @profnandaa. I've removed because the previous rule was blocking numbers starting with 1 after 9 and those are valid brazilian numbers, so I've replaced for invalid numbers starting with 0 after 9. In [this PR](https://github.com/validatorjs/validator.js/pull/1407/files#diff-0c277c9df3dc2342bb81f530112aba44ba8a3ab6724e1bad74acb99d1b9eca6cR5209-R5215)  from last year it was added, but they aren't invalid numbers.